agent/acs/handler/attach_eni_handler_test.go

```diff
@@ -28,9 +28,9 @@ import (
 )
 
 const (
-	eniMessageId = "123"
-	randomMAC    = "00:0a:95:9d:68:16"
-	waitTimeout  = 10
+	eniMessageId      = "123"
+	randomMAC         = "00:0a:95:9d:68:16"
+	waitTimeoutMillis = 10
```

Comment:
I understand that but this name is just confusing me. 
I would prefer `waitTimeout` or `waitTimeoutMs`.